flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -102,10 +102,11 @@ public void setUp() throws Exception {
         env.generateSequence(1, 1000).output(new DiscardingOutputFormat<>());
         plan = env.createProgramPlan();
 
-        final int freePort = NetUtils.getAvailablePort();
         config = new Configuration();
         config.setString(JobManagerOptions.ADDRESS, "localhost");
-        config.setInteger(JobManagerOptions.PORT, freePort);
+        try (NetUtils.Port port = NetUtils.getAvailablePort()) {
```

Comment:
This should be `port = NetUtils.getAvailablePort()` where port is a field in the test class. In a `@After` you'd then call `port.close()`.

Reply:
Thanks. I'll do it.